auth/authz.go

```diff
@@ -304,15 +305,15 @@ func CreatePolicy(ctx context.Context, clientsEndpoint string, clientID string,
 			"policy": policy,
 			"err":    err.Error(),
 		}, "Unable to marshal keyclaok policy struct")
-		return "", errors.NewInternalError("unable to marshal keyclaok policy struct " + err.Error())
+		return "", errors.NewInternalError(errs.Wrap(err, "unable to marshal keyclaok policy struct"))
```

Comment:
Fixed in 9c46e4c3ad09b91df07f303d1b823e486d3ff2d5, cfc22a590dc7bf579e4c09b21e401b54e6a952b3, and 671a4503471f054e76ee066940e0b74c59e6e312